nominees/our-world-in-data.json

```diff
@@ -3,47 +3,134 @@
+    },
+    {
+      "SDGNumber": 6,
+      "evidenceText": "We track and visualize the data relevant to this SDG.",
+      "evidenceURL": "https://sdg-tracker.org/water-and-sanitation"
+    },
+    {
+      "SDGNumber": 7,
+      "evidenceText": "We track and visualize the data relevant to this SDG.",
+      "evidenceURL": "https://sdg-tracker.org/energy"
+    },
+    {
+      "SDGNumber": 8,
+      "evidenceText": "We track and visualize the data relevant to this SDG.",
+      "evidenceURL": "https://sdg-tracker.org/economic-growth"
+    },
+    {
+      "SDGNumber": 9,
+      "evidenceText": "We track and visualize the data relevant to this SDG.",
+      "evidenceURL": "https://sdg-tracker.org/infrastructure-industrialization"
+    },
+    {
+      "SDGNumber": 10,
+      "evidenceText": "We track and visualize the data relevant to this SDG.",
+      "evidenceURL": "https://sdg-tracker.org/inequality"
+    },
+    {
+      "SDGNumber": 11,
+      "evidenceText": "We track and visualize the data relevant to this SDG.",
+      "evidenceURL": "https://sdg-tracker.org/cities"
+    },
+    {
+      "SDGNumber": 12,
+      "evidenceText": "We track and visualize the data relevant to this SDG.",
+      "evidenceURL": "https://sdg-tracker.org/sustainable-consumption-production"
+    },
+    {
+      "SDGNumber": 13,
+      "evidenceText": "We track and visualize the data relevant to this SDG.",
+      "evidenceURL": "https://sdg-tracker.org/climate-change"
+    },
+    {
+      "SDGNumber": 14,
+      "evidenceText": "We track and visualize the data relevant to this SDG.",
+      "evidenceURL": "https://sdg-tracker.org/oceans"
+    },
+    {
+      "SDGNumber": 15,
+      "evidenceText": "We track and visualize the data relevant to this SDG.",
+      "evidenceURL": "https://sdg-tracker.org/biodiversity"
+    },
+    {
+      "SDGNumber": 16,
+      "evidenceText": "We track and visualize the data relevant to this SDG.",
+      "evidenceURL": "https://sdg-tracker.org/peace-justice"
+    },
     {
       "SDGNumber": 17,
-      "evidenceText": "Related to Target 17.6 - Enhances knowledge sharing, specifically making this data available to individuals globally.",
-      "evidenceURL": "https://en.wikipedia.org/wiki/Our_World_in_Data"
+      "evidenceText": "We track and visualize the data relevant to this SDG.",
+      "evidenceURL": "https://sdg-tracker.org/global-partnerships"
     }
   ],
   "sectors": [
-    "Digital/Data/Tech",
-    "Fair & Responsible Media",
-    "Media",
-    "Research"
+    "Digital/Data/Tech"
   ],
   "type": [
+    "content",
+    "software",
     "data"
   ],
   "repositories": [
     {
-      "name": "main",
-      "url": ""
+      "name": "owid-grapher",
+      "url": "https://github.com/owid/owid-grapher"
+    },
+    {
+      "name": "owid/covid-19-data",
+      "url": "https://github.com/owid/covid-19-data"
+    },
+    {
+      "name": "owid/sdg-tracker.org",
+      "url": "https://github.com/owid/sdg-tracker.org"
```

Comment:
This appears to be unlicensed, or proprietary code under default copyright laws. We should check if upstream is willing to distribute this work under an open source license.

Reply:
The [Grapher codebase](https://github.com/owid/owid-grapher) is already licensed under MIT (very open). But the OWID website can’t be open licensed as a whole, because they are using data from various providers that are included in the repos (can’t relicense third-party data files). Something similar happens with the Covid repo - they’re happy to distribute the scripts under open license but we can’t license the data from third parties.

Cc: @jwflory

Reply:
Below is the response from the team:

```
Our [Grapher codebase](https://github.com/owid/owid-grapher) is already licensed under MIT (very open). But the OWID website can’t be open licensed as a whole, because we are using data from various providers that are included in the repos (we can’t relicense third-party data files). Something similar happens with the Covid repo - we’re happy to distribute the scripts under open license but we can’t license the data from third parties. 

We’ll be re-organising some of this, and we plan to do more around IP and license management (both in GitHub and in our website). But this will take several months - I realise this may not fit neatly the timeline and requirements you have at DPGA at the moment.
```